### ISSUE TITLE
Validation시 net.eval()사용하면 Agent가 아주 많이 멍청해짐

### ISSUE THREAD

**Issue by wolfrev0** [0xC0DEF/A2C#1]
https://www.google.com/search?q=pytorch+train+mode+eval+mode+performance+difference&newwindow=1&sxsrf=ALeKk03VuPPXTfbJa01dqimE5e6mUyMprQ%3A1617054264393&source=hp&ei=OEpiYK7cFcKNr7wP8OmlwAg&iflsig=AINFCbYAAAAAYGJYSFUyI_KURBUoVEYkf8uokyWn-dhY&oq=pytorch+train+mode+eval+mode+performance+difference&gs_lcp=Cgdnd3Mtd2l6EAMyBwghEAoQoAE6BAgjECc6BQgAEJECOgQIABBDOgcIIxDqAhAnOgIIADoCCC46BQgAEMsBOggIABAWEAoQHjoGCAAQFhAeOgcIABCHAhAUOgQIABANOgUIIRCgAToECCEQFVCaBliNzAFg-8wBaAZwAHgAgAGmAYgB-DeSAQQwLjU2mAEAoAEBqgEHZ3dzLXdperABCg&sclient=gws-wiz&ved=0ahUKEwiu2InIvNbvAhXCxosBHfB0CYgQ4dUDCAY&uact=5
임시로 Train.ipynb코드를 .train()사용하게 했지만 찝찝하다.

**Comment by wolfrev0** [comment]
Batch Normalization을 전부다 제거하니까 해결됨. 배치크기가 너무 작아서 그런가? 일단 학습이 잘 되니까 정규화 및 학습가속 의미를 가지는 배치노말은 없어도 되겠다.
